build.gradle

```diff
@@ -265,6 +265,8 @@ dependencies {
   compile 'com.jolbox:bonecp'
   compile 'org.apache.openjpa:openjpa'
   compile 'org.postgresql:postgresql'
+  // compile 'com.oracle:ojdbc8'
```

Comment:
Thinking this shouldn't even be here as we ever use a maven repos and include the driver

Reply:
Yes! Fixed!

Reply:
Should we remove the line in version.gradle as well?

Reply:
yep

Reply:
done